core-bundle/src/Resources/contao/library/Contao/Search.php

```diff
@@ -177,6 +177,17 @@ public static function indexPage($arrData)
 		// Calculate the checksum
 		$arrSet['checksum'] = md5($arrSet['text']);
 
+		$blnIndexExists = (bool) $objDatabase
```

Comment:
Why the cast? The if condition does not require type safety.

Reply:
Should the variable name still be `$blnIndexExists` then? Or `$strIndexExists` instead?

Reply:
It actually does not matter. I guess we have been using `$bln` for any bool-ish value.

Will MySQL return `0` and `1` or `'0'` and `'1'`?

Reply:
`'0'` and `'1'`

Reply:
I would still use `$bln` I guess.

Reply:
Done in e3310bbc84f8d54dbbd8c8d956ba6a651a070aa7